scripts/helper_functions.sh

```diff
@@ -100,4 +100,53 @@ init_ssh_agent() {
 
 commits_count() {
   curl -s --HEAD --header "PRIVATE-TOKEN: $SECRET_GITLAB_ACCESS_TOKEN" "${CI_SERVER_URL}/api/v4/projects/$CI_PROJECT_ID/repository/commits?per_page=1&ref_name=$CI_COMMIT_REF_NAME" | grep x-total: | cut -d " " -f2
-}
\ No newline at end of file
+}
+
+gcm_write_log() {
+  local log_name=$1
+  local payload=$2
+
+  gcloud logging write $log_name "$payload" --project=$GCP_PROJECT_ID --payload-type=json
+}
+
+gcm_write_metric() {
+  # metric format documentation: https://cloud.google.com/monitoring/custom-metrics/creating-metrics#writing-ts
+  local metric_type=$1
+  local labels=$2
+  local value=$3
+  local value_type=$4
+  local metric_kind=${5:-GAUGE}
+
+  curl -sS \
+    -H "Authorization: Bearer $(gcloud auth application-default print-access-token)" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/json; charset=utf-8' \
+    "https://monitoring.googleapis.com/v3/projects/$GCP_PROJECT_ID/timeSeries" \
```

Comment:
we had discussions with Sandra where to use `${variable}` instead of plain `$variable`. We had decided that once the variable is part of a larger text, it should be `${variable}` for better readability. I see that nobody does it like this anymore :crying_cat_face:

Reply:
Sounds reasonable, I like it :) Changed